src/pkg/cli/compose/loader.go

```diff
@@ -79,10 +79,7 @@ func (c *Loader) LoadProjectName(ctx context.Context) (string, error) {
 	return project.Name, nil
 }
 
-func (c *Loader) LoadProject(ctx context.Context) (*Project, error) {
-	if c.cached != nil {
-		return c.cached, nil
-	}
+func (c *Loader) LoadProjectOptions(ctx context.Context) (*cli.ProjectOptions, error) {
```

Comment:
This doesn't load so perhaps Get* is a better prefix. But tbh you're pretty much just using `newProjectOptions` as-is, so maybe just make that one public? Can `newProjectOptions` output logs? If so, we can move the `logrus` stuff into that func.